packages/apidom-ls/src/services/validation/linter-functions.ts

```diff
@@ -1081,9 +1081,22 @@ export const standardLinterfunctions: FunctionItem[] = [
           isArrayElement(element.parent) &&
           includesClasses(['path-item-parameters'], element.parent);
 
-        if (!isInPathItemElement) return true;
+        const isInOperationElement =
+          isArrayElement(element.parent) &&
+          includesClasses(['operation-parameters'], element.parent);
+
+        if (!isInPathItemElement && !isInOperationElement) return true;
+
+        let pathItemElement = {} as Element;
```

Comment:
Would it be possible to simplify into following form?

```js
const pathItemElement: Element | undefined = isInOperationElement
  ? element.parent?.parent?.parent?.parent?.parent
  : element.parent?.parent?.parent;

if (pathItemElement?.element !== 'pathItem') return true;
```

NOTE that I've also made traversing the parents safe by optional chaining, which should eliminate future corner cases.

Reply:
Yep it works fine 👍🏻 I changed the implementation.